### DISCUSSION TITLE
.HEIC support - incorrect colors after save as .JPG

### DISCUSSION THREAD

**Discussion by ukushu** [dlemstra/Magick.NET#808] [Help]
### Description
.HEIC support - incorrect colors after save as .JPG

### Steps to Reproduce
```
using (MagickImage magImg = new MagickImage(path))
{
    magImg.Format = MagickFormat.Jpg;
    magImg.Quality = 90;
    magImg.SetProfile(ColorProfile.SRGB);
    magImg.Write($"{saveToFolder}\\{filename}.jpg");
}
```

file sample:

https://www.heicconvert.com/images/examples/citron.HEIC

result looks like this

[![введите сюда описание изображения][1]][1]

  [1]: https://i.stack.imgur.com/DAaw6.jpg

### System Configuration

Magick.NET-Q16-AnyCPU - 7.22.2.2
Windows 10 x64 2004 build 19041.685

**Comment by dlemstra** [comment]
It seems that adding that profile messes up the colors. I get the correct colors with `magImg.ColorSpace = ColorSpace.sRGB;`.
